### PR TITLE
BAU - Fix tab indexing on the activity tab to include the progressive…

### PR DESCRIPTION
**Pull request by aaroncarroll82**
… disclosure elements

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html [hunk 1]
```diff
@@ -82,7 +82,7 @@
         }
         @if(request.hasPermission(Permission.MOVE_CASE_BACK_TO_QUEUE) && reassignCaseStatuses.contains(c.status)){
             <span class="column-one-third mt-1">
-                <a tabindex="@{startAtTabIndex+13}" id="reassign-queue-link" href="@ReassignCaseController.showAvailableQueues(c.reference, request.uri)">
+                <a tabindex="@{startAtTabIndex+213}" id="reassign-queue-link" href="@ReassignCaseController.showAvailableQueues(c.reference, request.uri)">
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html]: 213?  Are you sure?

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html [hunk 2]
```diff
@@ -114,15 +119,17 @@
...
                             )
                         }
                         case EventType.QUEUE_CHANGE => {
                             val body = event.details.asInstanceOf[QueueChange]
                             progressive_disclosure(
                                 id = s"activity-events-row-$index",
                                 summary = s" ${assignee(event.operator, true)} moved this case to the ${body.to.flatMap(to => queues.find(to == _.id)).map(_.name).getOrElse("unknown")} queue",
-                                body = body.comment.map(Html(_))
+                                body = body.comment.map(Html(_)),
+                                Some(startAtTabIndex + (index*2))
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html]: is that (Some(start...) the tab index property?  If so, why is it not labelled?  "useTabIndex = Some...."

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html [hunk 3]
```diff
@@ -49,27 +49,29 @@
...
                         }
                         case EventType.APPEAL_ADDED => {
                             val body = event.details.asInstanceOf[AppealAdded]
                             if(body.appealType == AppealType.REVIEW) {
                                 progressive_disclosure(
                                     id = s"activity-events-row-$index",
                                     summary = s"Review added with status ${AppealStatus.formatReview(body.appealStatus)}",
-                                    body = body.comment.map(Html(_))
+                                    body = body.comment.map(Html(_)),
+                                    Some(startAtTabIndex + (index*2))
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html]: label "useTabIndex" param - lots of examples of this

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html (hunk 1 above)
Comment (aaroncarroll82) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html]: Gives us a possible 200 to use for any events (so 100 events), as this appears afterwards.

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html (hunk 2 above)
Comment (aaroncarroll82) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html]: good point, will fix

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html (hunk 3 above)
Comment (aaroncarroll82) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html]: as above, will fix this

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html (hunk 1 above)
Comment (aaroncarroll82) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html]: I made it a nicer round number anyway

**Pull request merged**